libqtile/core/manager.py

```diff
@@ -288,9 +286,15 @@ def _process_screens(self) -> None:
             if not self.current_screen:
                 self.current_screen = scr
 
-            scr._configure(
-                self, i, x, y, w, h, self.groups[i],
-            )
+            if len(self.groups) < i + 1:
+                name = f"group_{i + 1}"
```

Comment:
Why not just `str(i)`? Because numbers are common names and you don't want collision? Feels like a hack to me, but I don't have much better to offer. :(

Reply:
Yep exactly, just reducing the change of a collision. I figure as well that if someone is naming their groups "group_1, group_2" etc, then they certainly will not have a "group_5" when there are fewer than 5 groups, so that is much less likely than someone having a group just named "5". So maybe bit of a hack, buuuuuuuuuut :)

Reply:
What about "autogen_1"?

Reply:
sure thing, changed